playwright/pages/profile/delegationPage.ts

```diff
@@ -172,3 +221,41 @@ export class revokeRights {
     }
   }
 }
+
+export class instantiateResource {
+  constructor(public page: Page) {}
+
+  async instantiateApp(appReportee: string) {
+    // Go to the app URL
+    await this.page.goto(process.env.APP_URL as string);
+    try {
+      const headingVisible = await this.page
+        .waitForSelector('role=heading[name="Hvem vil du sende inn for?"]', { timeout: 5000 })
+        .then(() => true)
+        .catch(() => false);
+
+      if (headingVisible) {
+        await this.page.getByPlaceholder('Søk etter aktør').click();
```

Comment:
Kan du vurdere å bruke tekstene som ligger i frontendrepoet fra før av her i steden for å bruke hardkodet tekst? (Har ikke sjekket om den faktisk ligger i den konstant-listen, så du får nesten sjekke om det er verdt det)